src/DynamoCoreWpf/Properties/Resources.Designer.cs

```diff
@@ -3672,7 +3717,7 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Help....
+        ///   Looks up a localized string similar to Help.
```

Comment:
@OliverEGreen After some discussion, I think we need to re-evaluate the `...` decision to keep or not based on https://docs.microsoft.com/en-us/windows/win32/uxguide/cmd-menus#using-ellipses. I will defer to @Jingyi-Wen on this